projects/batfish/src/test/java/org/batfish/bdp/BdpDataPlanePluginTest.java

```diff
@@ -0,0 +1,90 @@
+package org.batfish.bdp;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Map;
+import java.util.SortedMap;
+import java.util.TreeMap;
+import org.apache.commons.collections4.map.LRUMap;
+import org.batfish.common.BatfishLogger;
+import org.batfish.config.Settings;
+import org.batfish.config.Settings.EnvironmentSettings;
+import org.batfish.config.Settings.TestrigSettings;
+import org.batfish.datamodel.Configuration;
+import org.batfish.datamodel.ConfigurationFormat;
+import org.batfish.datamodel.DataPlane;
+import org.batfish.datamodel.Interface;
+import org.batfish.datamodel.collections.BgpAdvertisementsByVrf;
+import org.batfish.datamodel.collections.RoutesByVrf;
+import org.batfish.main.Batfish;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/** Tests for {@link BdpDataPlanePlugin}. */
+public class BdpDataPlanePluginTest {
+
+  @Rule public TemporaryFolder _folder = new TemporaryFolder();
+
+  private static Configuration createConfiguration(String hostname, String interfaceName) {
+    Configuration config = new Configuration(hostname);
+    config.setConfigurationFormat(ConfigurationFormat.HOST);
+    config.getInterfaces().put(interfaceName, new Interface(interfaceName, config));
+    return config;
+  }
+
+  private Batfish initBatfish(SortedMap<String, Configuration> configurations) throws IOException {
```

Comment:
yes, that would be better I think. I will make it configurable through arguments to make it usable across various functions